nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractPutElasticsearch.java

```diff
@@ -55,10 +55,41 @@
 import java.util.stream.Collectors;
 
 public abstract class AbstractPutElasticsearch extends AbstractProcessor implements ElasticsearchRestProcessor {
+    static final Relationship REL_ORIGINAL = new Relationship.Builder()
+            .name("original")
+            .description("All flowfiles that are sent to Elasticsearch without request failures go to this relationship.")
+            .build();
+
+    static final Relationship REL_SUCCESSFUL = new Relationship.Builder()
+            .name("successful")
```

Comment:
Is there a particular reason for calling this `successful` as opposed to the much more common `success`?

Reply:
That, and I think renaming the relationship might impact other ES processors (I'll have to double check)